DEV.md

```diff
@@ -7,6 +7,11 @@ Building Scala CLI requires JVM 17 to work properly.
 In theory, our build is able to download and install for its own needs JVM 17 on some OSes however it may not work in
 Intellij / Metals out of the box.
 
+When working with IntelliJ make sure that under `File -> Project Structure`:
+- in `Project Settings/Project` `SDK` and `Language level` is set to **17**
+- in `Project Settings/Modules` all the modules have `Language level` set to **17**
+- in `Platform Settings/SDKs` only **Java 17** is visible
```

Comment:
```suggestion
When working with IntelliJ make sure that the project's Java is set correctly.
To confirm that, check under `File -> Project Structure`:
- in `Project Settings/Project` `SDK` and `Language level` is set to **17**
- in `Project Settings/Modules` all the modules have `Language level` set to **17**
- in `Platform Settings/SDKs` only **Java 17** is visible
Otherwise, some IDE features may not work correctly, i.e. the debugger might crash upon connection.
```
BTW I think this should be under the `IDE import` section, not `Requirements`